--- xqf-1.0.5/xqf.desktop.in~	2007-01-18 15:49:43.000000000 +0100
+++ xqf-1.0.5/xqf.desktop.in	2007-01-18 15:48:53.000000000 +0100
@@ -3,9 +3,11 @@
 _Name=XQF Game Server Browser
 _GenericName=Game Server Browser
 _Comment=Locate and connect to game servers
+_Comment[pl]=Narzędzie pozwalające lokalizować i łączyć się z serwerami gier
 Exec=xqf
 Icon=xqf_32x32.png
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GTK;Game;ActionGame;X-SuSE-Core-Game;
+Categories=GTK;Game;
+# vi: encoding=utf-8
